Update cross-tenant-mailbox-migration.md (#9510)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/cross-tenant-mailbox-migration.md

```

> [!NOTE]

> > The email address in the CSV file must be the one specified in the target tenant (e.g. userA@targettenant.onmicrosoft.com), not the source tenant.

Reviewer: ```suggestion
> The email address in the CSV file must be the one specified in the target tenant (for example, userA@targettenant.onmicrosoft.com), not the one in the source tenant.
```

>
> [For more information on the cmdlet click here](/powershell/module/exchange/new-migrationbatch)
>
> [For some example CSV file info click here](/exchange/csv-files-for-mailbox-migration-exchange-2013-help)

> Here is a minimal example CSV.

Reviewer: ```suggestion
The following is a minimal example CSV file:
```

This can be done before the migration is complete, but you should not assign a license prior to stamping the _ExchangeGuid_ attribute or the conversion of MailUser object to mailbox will fail and a new mailbox will be created instead. To mitigate this risk, it is best to wait until after the migration is complete, and assign licenses during the 30 day grace period.

### Can I use Azure AD Connect to sync users to the new tenant if I am keeping the on-prem Active Directory?

> Yes. It is possible to have two instances of Azure AD Connect syncronize to different tenants.
> However there are some things you need to be aware of.

Reviewer: ```suggestion
Yes. It is possible to have two instances of Azure AD Connect synchronize to different tenants.
However, there are some things you need to be aware of.
```

> - Preprovisioning the users accounts with the script provided in this articale should not be done. Instead a selective OU sync of the users in scrope for the migation can be performed to populate the target tenant - you will receive a warning about the UPN not matching during AAD Connect configuration.

Reviewer: ```suggestion
- Preprovisioning the user's accounts with the script provided in this article should not be done. Instead, a selective OU sync of the users in scope for the migration can be performed to populate the target tenant; you will receive a warning about the UPN not matching during Azure AD Connect configuration.
```

> - Depending on your current state of Hybrid Exchange, you will need to verify that the on-prem directory objects have the required attributes populated correctly (i.e. msExchMailboxGUID, proxyAddresses etc.) before attempting to sync to another tenant, or you will run into issues with double mailboxes and migration failures.

Reviewer: ```suggestion
- Depending on your current state of Hybrid Exchange, you need to verify that the on-prem directory objects have the required attributes (such as msExchMailboxGUID and proxyAddresses) populated correctly before attempting to sync to another tenant, or you will run into issues with double mailboxes and migration failures.
```

> - You will need to take some ekstra steps to manage UPN transitioning. Changing it on-prem once the migration has completed for a user, unless you are also moving the custom domain during a cut-over migration.

Reviewer: ```suggestion
- You need to take some extra steps to manage UPN transitioning, changing it on-prem once the migration has been completed for a user unless you are also moving the custom domain during a cut-over migration.
```
